qiskit/aqua/operators/evolutions/evolved_op.py

```diff
@@ -159,3 +159,7 @@ def log_i(self, massive: bool = False) -> OperatorBase:
     # pylint: disable=arguments-differ
     def to_instruction(self, massive: bool = False) -> Instruction:
         return self.primitive.to_matrix_op(massive=massive).exp_i()  # type: ignore
+
+    # def to_gate(self, massive: bool = False) -> Gate:
```

Comment:
This should use the `HamiltonianGate`, see the `to_circuit` (or `to_circuit_op`)

Reply:
Noted!